### PR TITLE
PWN-4733 - Fix verify errors states

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4733
https://p2pvalidator.atlassian.net/browse/PWN-4774

## Description of Work

Fix error states for SMS verify and phone verify

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/auth/ui/generalerror/timer/OnboardingGeneralErrorTimerFragment.kt [hunk 1]
```diff
@@ -36,17 +36,20 @@ class OnboardingGeneralErrorTimerFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
+        requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
+            returnToPhoneNumberEnter()
+        }
         binding.buttonToStartingScreen.setOnClickListener {
             returnToPhoneNumberEnter()
         }
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/ui/generalerror/timer/OnboardingGeneralErrorTimerFragment.kt:44]:
```suggestion
        binding.buttonToStartingScreen.setOnClickListener {
            returnToPhoneNumberEnter()
        }
        requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
            returnToPhoneNumberEnter()
        }
```
i saw we place backPressedDispatcher always in the end of the onViewCreate

File: app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterPresenter.kt [hunk 2]
```diff
@@ -77,6 +80,7 @@ class PhoneNumberEnterPresenter(
     override fun submitUserPhoneNumber(phoneNumber: String) {
         launch {
             try {
+                throw GatewayServiceError.SmsDeliverFailed(32053, "Test")
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterPresenter.kt:83]:
```suggestion
```

File: app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterPresenter.kt [hunk 3]
```diff
@@ -86,6 +90,10 @@ class PhoneNumberEnterPresenter(
                     view?.navigateToSmsInput()
                 }
             } catch (smsDeliverFailed: GatewayServiceError.SmsDeliverFailed) {
+                if (++submitUserPhoneTriesCount >= MAX_PHONE_NUMBER_TRIES) {
+                    view?.navigateToAccountBlocked()
+                    return@launch
+                }
                 Timber.i(smsDeliverFailed)
                 view?.showSmsDeliveryFailedForNumber()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterPresenter.kt:98]:
```suggestion
                Timber.i(smsDeliverFailed)
                if (++submitUserPhoneTriesCount >= MAX_PHONE_NUMBER_TRIES) {
                    view?.navigateToAccountBlocked()
                } else {
                    view?.showSmsDeliveryFailedForNumber()
                }
```

File: app/src/main/res/values/environment_config.xml [hunk 4]
```diff
@@ -12,7 +12,7 @@
     <string name="feeRelayerBaseUrl">https://fee-relayer.solana.p2p.org/</string>
     <string name="feeRelayerTestBaseUrl">https://test-solana-fee-relayer.wallet.p2p.org/</string>
 
-    <string name="torusBaseUrl">http://localhost</string>
+    <string name="torusBaseUrl">http://beardict.net</string>
```
Comment (gslevinkov) [inline app/src/main/res/values/environment_config.xml:15]: revert?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/res/values/environment_config.xml (hunk 4 above)
Comment (davran1001) [reply app/src/main/res/values/environment_config.xml:15]: i reverted it, but not added ((

**Pull request merged**